### PR TITLE
Fix test task manager config

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

**Comment by sopel39** [comment]
FYI: @dain

File: presto-main/src/test/java/com/facebook/presto/execution/TestTaskManagerConfig.java [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
-                .setInitialSplitsPerNode(Runtime.getRuntime().availableProcessors() * 4)
+                .setInitialSplitsPerNode(Runtime.getRuntime().availableProcessors() * 2)
                 .setSplitConcurrencyAdjustmentInterval(new Duration(100, TimeUnit.MILLISECONDS))
                 .setStatusRefreshMaxWait(new Duration(1, TimeUnit.SECONDS))
                 .setInfoUpdateInterval(new Duration(200, TimeUnit.MILLISECONDS))
                 .setVerboseStats(false)
                 .setTaskCpuTimerEnabled(true)
                 .setMaxWorkerThreads(Runtime.getRuntime().availableProcessors() * 2)
-                .setMinDrivers(Runtime.getRuntime().availableProcessors() * 4 * 2)
+                .setMinDrivers(Runtime.getRuntime().availableProcessors() * 4)
```
Comment (akshatnair) [inline presto-main/src/test/java/com/facebook/presto/execution/TestTaskManagerConfig.java]: change this to * 2 * 2
to reflect that it is basically doing twice the number of max worker threads

**Comment by sopel39** [comment]
FYI: @martint: this fixes Travis error in Master

**Comment by nileema** [comment]
This is fixed by #5274

**Pull request closed without merging**